.github/workflows/continous-integration.yml

```diff
@@ -2,8 +2,8 @@ name: Continuous Integration
 
 on:
   push:
-    branches:
-      - main
+    branches-ignore:
+      - nightly_docker
```

Comment:
Double-checking that this means the CI workflow doesn't run for nightly releases?

Reply:
This is only to test this PR. I skipped test run etc to make testing of workflow faster. I will revert back once i am done testing successful docker image push :)